zipkin-server/README.md

```diff
@@ -579,6 +579,16 @@ of known open source libraries on 128-bit trace identifiers.
 
 See `zipkin2.storage.StorageComponent.Builder` for even more details!
 
+## TLS/SSL
```

Comment:
added it at the end of the README since i don't think this is a prominent feature that everyone is looking for